src/main/java/ui/UI.java

```diff
@@ -143,6 +143,11 @@ private void login(boolean isBypassLogin) {
         getMainWindowHandle(mainStage.getTitle());
     }
 
+    private void openSampleBoard(){
```

Comment:
The sample board is opened as well as saved in the config file (when it is created and opened). Let's go ahead with `openAndSaveSampleBoard`.